src/scenes/home/Home.js

```diff
@@ -19,17 +35,53 @@ export class Home extends Component {
     headerTintColor: colors.white,
     headerTitleStyle: styles.text
   };
+  
+  constructor(props) {
+    super(props);
+    this.state = {
+      tasks: data,
+    };
+  }
+
+  toggleDone = (id) => {
+    const { tasks } = this.state;
+    const data = tasks.slice();
+    const task = data.find(x => x.id === id);
+    task.done = !task.done;
+    this.setState({
+      tasks: data,
+    });
+  }
+
+  markAsDone = (id) => {
+    const { tasks } = this.state;
+    const data = tasks.slice();
+    const task = data.find(x => x.id === id);
```

Comment:
this are the same that the above function, maybe you could use the same function..